cluster_test.go

```diff
@@ -2083,8 +2098,8 @@ func (s *S) TestDoNotFallbackToMonotonic(c *C) {
 	if !s.versionAtLeast(3, 0) {
 		c.Skip("command-counting logic depends on 3.0+")
 	}
-	if s.versionAtLeast(3, 4) {
-		c.Skip("failing on 3.4+")
+	if s.versionAtLeast(3, 2, 17) {
```

Comment:
Maybe best to move all version checks to the top of the method, before any mgo.Dial() calls?

Reply:
It's already at the top of the method ( `TestDoNotFallbackToMonotonic` ) but I agree that it's not clear in the git diff